test/library_test.rb

```diff
@@ -0,0 +1,35 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/book'
+require './lib/author'
+require './lib/library'
+
+class LibraryTest < Minitest::Test
+
+  def setup
+    @dpl = Library.new("Denver Public Library")
+    @charlotte_bronte = Author.new({first_name: "Charlotte", last_name: "Bronte"})
+    @harper_lee = Author.new({first_name: "Harper", last_name: "Lee"})
+  end
+
+  def test_it_exists
+    assert_instance_of Library, @dpl
+  end
+
+  def test_attributes
+    assert_equal "Denver Public Library", @dpl.name
+    assert_equal [], @dpl.books
+    assert_equal [], @dpl.authors
+  end
+
+  def test_it_adds_authors
+    @dpl.add_author(@charlotte_bronte)
+    @dpl.add_author(@harper_lee)
+    jane_eyre = @charlotte_bronte.write("Jane Eyre", "October 16, 1847")
+    professor = @charlotte_bronte.write("The Professor", "1857")
+    # villette = @charlotte_bronte.write("Villette", "1853")
```

Comment:
When you add an author to the library, all of that authors books are also added.  So, if charlotte writes 'Villette', I would expect the libraries books to include Villette and my expecation would look like: `assert_equal [jane_eyre, professor, villette, mockingbird], @dpl.books`

Because your expectation does not include `villette`, charlotte can't write it - because the library has _all_ the books she has written.